app/src/split/bluetooth/central.c

```diff
@@ -265,9 +267,14 @@ static uint8_t split_central_chrc_discovery_func(struct bt_conn *conn,
                             BT_UUID_DECLARE_128(ZMK_SPLIT_BT_CHAR_RUN_BEHAVIOR_UUID))) {
         LOG_DBG("Found run behavior handle");
         slot->run_behavior_handle = bt_gatt_attr_value_handle(attr);
+    } else if (!bt_uuid_cmp(((struct bt_gatt_chrc *)attr->user_data)->uuid,
+                            BT_UUID_DECLARE_128(ZMK_SPLIT_BT_UPDATE_LEDS_UUID))) {
+        LOG_DBG("Found update LEDs handle");
+        slot->update_leds_handle = bt_gatt_attr_value_handle(attr);
```

Comment:
`release_peripheral_slot()` clears `run_behavior_handle` to 0. Does that need to happen for `update_leds_handle` as well?